_data/api-swagger.yaml

```diff
@@ -256,9 +256,25 @@ definitions:
                 - `*Emphasis*` / `_Emphasis_`
                 - `**Stong**` / `__Strong__`
                 - `1. Ordered` and `- Unordered` lists
+            Authentications:
+              description: |
+                List of authentications that the signer has to authenticate with.
+                Authentications must be performed before the document(s) can be viewed.
+                The order in which the authentications are provided determine in which order the signer will have to perform the specified method.
+
+                You **must** explicitly specify the API-version when using this feature.
+                This is done with the header: 'Accept: application/vnd.signhost.v1+json'.
+
+                The following authentications are supported:
+                - DigiD
+                - PhoneNumber
+              type: array
+              items:
+                $ref: '#/definitions/Verification'
```

Comment:
Shouldn't we create a proper authentication model, instead of misusing the verifications.
Because If you would click through we would allow every verification as authentication this way.

Reply:
It states:

                The following authentications are supported:
                - DigiD
                - PhoneNumber

Reply:
Yes, that's correct. But since it links to the verification model there's a chance the reader gets confused, because if they would click through on the `array of type #/definitions/Verification` link in authentications they will be taken to the verification definition.

Which has more options and states that specific options needs to be provided as the last value, but this isn't the case for authentications and most of them aren't even supported as such.

The text and distinction between authentications and verifications might be obvious to us, but there shouldn't be any possibility for the reader to interpret the documentation wrong. The reader most-likely has never used the API before and reads the documentation for the first time.

I'm fine with leaving it like this for the time being, if @MrJoe agrees. But I do think we should spit this up in the near future.

Reply:
I think we should split this into a separate Authentication model as well. Keep in mind this is a swagger / openapi specifications file so in theory it should be possible to auto generate a client when this swagger file in json format is provided and without the split it would list Verifications as an authentication which is definitely not what we intended.